### PR TITLE
sql-16: create a profile menu component

### PR DESCRIPTION
**Pull request by IlliaLop**

### CODE REVIEW

**Comment by IlliaLop** [comment]
image

**Comment by Anatolii-Ivanov** [comment]
Update yarn.lock and move user icon to the bottom of sidebar, please

**Comment by IlliaLop** [comment]
> Update yarn.lock and move user icon to the bottom of sidebar, please

image

File: frontend/src/app/shared/components/profile-menu/profile-menu.component.sass [hunk 1]
```diff
@@ -0,0 +1,50 @@
+.user-info
+    display: flex
+    align-items: center
+    font-size: 12px
+    color: #6c757d
```
Comment (VladTkach) [inline frontend/src/app/shared/components/profile-menu/profile-menu.component.sass]: move color to global variable

File: frontend/src/app/shared/components/sidebar/sidebar.component.sass [hunk 2]
```diff
@@ -19,15 +19,19 @@
         height: 16px
         padding: 4px 2px
         margin-top: 22px
+        cursor: pointer
```
Comment (VladTkach) [inline frontend/src/app/shared/components/sidebar/sidebar.component.sass]: you can add prop such a curcor: pointer or margin-top just to img instead a couple classes

**Review by VladTkach — DISMISSED** [review]

File: frontend/src/assets/logout_icon.png
Comment (Anatolii-Ivanov) [inline frontend/src/assets/logout_icon.png:1]: Please, use SVGs instead

File: frontend/src/app/shared/components/profile-menu/profile-menu.component.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { Component } from '@angular/core';
+
+@Component({
+    selector: 'app-profile-menu',
+    templateUrl: './profile-menu.component.html',
+    styleUrls: ['./profile-menu.component.sass'],
+})
+export class ProfileMenuComponent {
+    // eslint-disable-next-line no-undef,no-empty-function,@typescript-eslint/no-useless-constructor,@typescript-eslint/no-empty-function
+    constructor() { }
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/shared/components/profile-menu/profile-menu.component.ts]: Remove constructor if it's not used

**Review by VladTkach — APPROVED** [review]

File: frontend/src/app/shared/components/profile-menu/profile-menu.component.html [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    <mat-icon class="icon" aria-hidden="true">
+        <img src="assets/settings_icon.svg" alt="Settings Icon">
+    </mat-icon>
+    Settings
+</button>
+<button mat-menu-item class="menu-item logout">
+    <mat-icon class="icon logout-icon" aria-hidden="true" style="color: #6f42c1;">
+        <img src="assets/logout_icon.svg" alt="Logout Icon">
+    </mat-icon>
+    Log out
```
Comment (VladTkach) [inline frontend/src/app/shared/components/profile-menu/profile-menu.component.html:25]: Can you and log out method that delete access tokens and also redirect to login?

**Review by Anatolii-Ivanov — APPROVED** [review]
For me, it's ok

**Pull request merged**